pkg/apis/eksctl.io/v1alpha5/validation.go

```diff
@@ -236,6 +236,12 @@ func ValidateNodeGroup(i int, ng *NodeGroup) error {
 		return err
 	}
 
+	if ng.CPUCredits != nil {
```

Comment:
sorry big nitpick, this check should go inside `validateCPUCredits` IMO, so it's always called like `validateInstancesDistribution` above.